Tools/Frame_params/Solo_AC36.param

```diff
@@ -0,0 +1,133 @@
+# NOTE: 1/8/2018: Required parameters for a 3DR Solo with ArduCopter 3.6. Not for use on ArduCopter 3.5!
+ACCEL_Z_I,1.5
+ACCEL_Z_P,0.75
+ACRO_RP_P,4
+ACRO_THR_MID,0.4
+ACRO_YAW_P,1
+AHRS_ORIENTATION,12
+ANGLE_MAX,2000
+ARMING_CHECK,7166
+ATC_ACCEL_P_MAX,36000
+ATC_ACCEL_R_MAX,36000
+ATC_ACCEL_Y_MAX,7000
+ATC_ANG_PIT_P,10
+ATC_ANG_RLL_P,9
+ATC_ANG_YAW_P,6
+ATC_RAT_PIT_D,0.006
+ATC_RAT_PIT_IMAX,0.444
+ATC_RAT_PIT_P,0.167
+ATC_RAT_RLL_D,0.005
+ATC_RAT_RLL_I,0.12
+ATC_RAT_RLL_IMAX,0.444
+ATC_RAT_RLL_P,0.086
+ATC_RAT_YAW_FILT,5
+ATC_RAT_YAW_I,0.05
+ATC_RAT_YAW_IMAX,0.222
+ATC_RAT_YAW_P,0.5
+ATC_SLEW_YAW,8000
+BATT_CAPACITY,5200
+BATT_MONITOR,5
+BATT_SERIAL_NUM,0
+BRD_IMU_TARGTEMP,45
+BRD_SAFETY_MASK,16368
+BRD_SAFETYENABLE,0
+BRD_SER1_RTSCTS,1
+CIRCLE_RADIUS,3000
+COMPASS_ORIENT,38
+COMPASS_ORIENT2,0
+COMPASS_ORIENT3,0
+COMPASS_PRIMARY,0
+COMPASS_TYPEMASK,15392
+COMPASS_USE,1
+COMPASS_USE2,1
+COMPASS_USE3,0
+DISARM_DELAY,8
+FENCE_ACTION,0
+FENCE_ALT_MAX,122
+FENCE_ENABLE,1
+FENCE_RADIUS,60
+FENCE_TYPE,1
+FLTMODE1,5
+FLTMODE2,5
+FLTMODE3,5
+FLTMODE4,5
+FLTMODE5,5
+FLTMODE6,5
+FRAME_CLASS,1
+FRAME_TYPE,1
+FS_BATT_ENABLE,2
+FS_BATT_MAH,520
+FS_BATT_VOLTAGE,14
+FS_EKF_ACTION,2
+FS_THR_VALUE,950
+GND_EFFECT_COMP,1
+GPS_HDOP_GOOD,200
+GPS_NAVFILTER,6
+INS_ACC_BODYFIX,3
+INS_FAST_SAMPLE,3
+INS_TRIM_OPTION,2
+LAND_SPEED,45
+LAND_SPEED_HIGH,220
+LOG_BACKEND_TYPE,3
+LOG_DISARMED,1
+LOG_FILE_BUFSIZE,8
+LOG_FILE_DSRMROT,1
+MNT_ANGMAX_TIL,0
+MNT_ANGMIN_TIL,-9000
+MNT_RC_IN_TILT,6
+MNT_TYPE,2
+MOT_BAT_CURR_MAX,50
+MOT_BAT_VOLT_MAX,16.8
+MOT_BAT_VOLT_MIN,12
+MOT_PWM_MAX,1869
+MOT_PWM_MIN,1000
+MOT_SPIN_ARM,0.08
+MOT_SPIN_MIN,0.12
+MOT_SPOOL_TIME,1
+MOT_THST_EXPO,0.8
+NTF_OREO_THEME,1
+PHLD_BRAKE_ANGLE,2000
+PHLD_BRAKE_RATE,6
+PILOT_ACCEL_Z,100
+PILOT_SPEED_UP,133
+PILOT_SPEED_DN,250
+PILOT_THR_BHV,7
+PILOT_THR_FILT,2
+PILOT_TKOFF_ALT,214
+PILOT_TKOFF_DZ,250
+RC_FEEL_RP,20
+RC1_DZ,20
+RC1_MAX,2000
+RC1_MIN,1000
+RC2_DZ,20
+RC2_MAX,2000
+RC2_MIN,1000
+RC3_MAX,1890
+RC3_MIN,1008
+RC3_TRIM,1450
+RC4_DZ,20
+RC4_MAX,2000
+RC4_MIN,1000
+RC6_MAX,1520
+RC6_MIN,1000
+RC6_TRIM,1000
+RC7_MAX,2000
+RC7_MIN,1000
+RTL_ALT,4500
+RTL_CLIMB_MIN,1000
+RTL_SPEED,800
+SERIAL1_BAUD,921
+SERIAL4_BAUD,230
+SERIAL4_PROTOCOL,1
+SERVO1_FUNCTION,33
+SERVO2_FUNCTION,34
+SERVO3_FUNCTION,35
+SERVO4_FUNCTION,36
+THR_DZ,10
+VEL_XY_P,1.4
+WP_YAW_BEHAVIOR,3
+WPNAV_LOIT_MAXA,229
+WPNAV_LOIT_MINA,108
+WPNAV_SPEED,300
```

Comment:
wpnav-speed of only 3m/s?  I'm happy if you're happy of course, but pretty slow!